docs/configuration/events/monster-events.rst

```diff
@@ -77,7 +77,15 @@ sta                 Stamina IV of the monster.
 base_catch          Base catch rate of the monster.
 great_catch         Catch rate of the monster when using a super ball.
 ultra_catch         Catch rate of the monster when using a ultra ball.
-=================== =========================================================
+base_catch          Catch rate of the monster when using a poke ball.
```

Comment:
base_catch 2x ? its already on top of the list here lol

Reply:
Fixed in dev, thanks for the heads up.